hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/upgrade/UpgradeException.java

```diff
@@ -112,7 +112,9 @@ public String toString() {
     REMOVE_UPGRADE_TO_LAYOUT_VERSION_FAILED,
     UPDATE_LAYOUT_VERSION_FAILED,
     LAYOUT_FEATURE_FINALIZATION_FAILED,
-    PREFINALIZE_STATE_VALIDATION_FAILED,
-    FIRST_UPGRADE_START_ACTION_FAILED;
+    PREFINALIZE_ACTION_VALIDATION_FAILED,
+    FIRST_UPGRADE_START_ACTION_FAILED,
+
```

Comment:
nit: do we need this newline here?

Reply:
Fixed (removed all newlines in between the result codes :))